api/src/main/java/marquez/api/DatasetResource.java

```diff
@@ -143,12 +144,16 @@ public Response listVersions(
   @Produces(APPLICATION_JSON)
   public Response list(
       @PathParam("namespace") NamespaceName namespaceName,
+      @QueryParam("pattern") @DefaultValue("%") String pattern,
```

Comment:
Although this is a `pattern`, query param names like `search` or `searchBy` seem to be more descriptive. In future we may replace like pattern with other search engines.

Reply:
Yeah, I can change it!